Apps/AdminWebClient/src/ClientApp/src/views/Stats.vue

```diff
@@ -46,10 +46,47 @@
 import { Component, Vue } from "vue-property-decorator";
 import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import container from "@/plugins/inversify.config";
+import { ICsvExportService } from "@/services/interfaces";
 @Component
 export default class StatsView extends Vue {
-    private downloadUserInfoCSV() {}
-    private downloadUserNotesCSV() {}
-    private downloadUserCommentsCSV() {}
+    private mounted() {
+        this.csvExportService = container.get(
+            SERVICE_IDENTIFIER.CsvExportService
+        );
+    }
+    private csvExportService!: ICsvExportService;
+    private downloadUserInfoCSV() {
+        this.csvExportService
+            .getUserProfiles()
+            .then((csvData: any) => {
+                console.log("user profiles retrieved.");
```

Comment:
at this section, the promise should be resolved or return a promise for the callee of this service

Reply:
right, as real service doesn't return anything at this time, to process further I need to debug this one. btw, not sure if presenting this file download is a part of this vue service PR. Regardless, some works need to be done here.